mysql-test/lib/My/SafeProcess/safe_process.cc

```diff
@@ -318,11 +318,6 @@ int main(int argc, char* const argv[] )
     if (nocore)
       setlimit(RLIMIT_CORE, 0, 0);
 
-    /*
-      mysqld defaults depend on that. make test results stable and independent
-      from the environment
-    */
-    setlimit(RLIMIT_NOFILE, 1024, 1024);
 
     // Signal that child is ready
```

Comment:
Please also remove empty line above. We don't normally add double empty lines within functions.